configure.sh

```diff
@@ -157,3 +192,9 @@ build --action_env ANDROID_SDK_API_LEVEL="29"
 build --action_env ANDROID_SDK_HOME="/tmp/lce_android"
 
 EOM
+
+if is_windows; then
+  write_to_bazelrc "build --config=v2 --config=windows"
+else
+  write_to_bazelrc "build --config=v2 --config=noaws --config=nogcp --config=nohdfs --config=nonccl"
+fi
```

Comment:
For some reason setting `--config=noaws --config=nogcp --config=nohdfs --config=nonccl` fails due to bazel config conflicts.